spec/request/v1/api_provides_sign_up_functionality_spec.rb

```diff
@@ -0,0 +1,98 @@
+RSpec.describe 'User Registration', type: :request do
+  let(:headers) { { HTTP_ACCEPT: 'application/json' } }
+
+  describe 'User inputs valid credentials' do
+
+    before do
+      post '/v1/auth',
```

Comment:
auth needs to be outside of the v1 namespace

Reply:
done